encryption/.post-create

```diff
@@ -0,0 +1,20 @@
+To begin development, install Docker:
```

Comment:
Do we have to have all of these files for every sample? If we want to change the content, will we be updating them all? Same goes for the dependencies in package.json, are we going to have to keep dozens of dependency files up to date? Neither Java nor Go samples have to.

Reply:
We have a lot of config files like this that are duplicated in git. Some have modifications, in which case they're excluded from the pre-commit copy script: 

https://github.com/temporalio/samples-typescript/blob/main/.scripts/copy-shared-files.mjs

Separate `package.json`s so they work standalone, eg via our command-line tool, `npx @temporalio/create@latest --sample hello-world`

Reply:
Better design would be for each directory have a `.copy-config` file, rather than listing them in the script